### ISSUE TITLE
Usage with webpack?

### ISSUE THREAD

**Issue by mikebronner** [tempusdominus/bootstrap-3#3]
Hi, I am trying to get the plugin installed using webpack, and it seems to work. I have

```
global.moment = require('moment');
require('tempusdominus-bootstrap-4');
```

And everything runs without any JS errors, but I'm not seeing the plugin registered in jQuery. Any suggestions? :)

**Comment by chriseenberg** [comment]
Did you get it working?

**Comment by mikebronner** [comment]
@chriseenberg let me get back to you on that ... I think I got the Bootstrap 3 version working with a caveat, I'll have to pull up the project again next week and take a look.

**Comment by theMadness** [comment]
The way I have it working is having this in package.json

```
  "dependencies": {
    "tempusdominus-bootstrap-4": "https://github.com/tempusdominus/bootstrap-4.git"
  }
```

in the js then I use 

```
global.moment = require('moment');
require('tempusdominus-bootstrap-4');
```

`$('.datetimepicker').datetimepicker();` should work now.

**Comment by chriseenberg** [comment]
Thanks for this. I will give it a go :)

**Comment by ejlocop** [comment]
any update @chriseenberg, 

it seems like doing 
```javascript
global.moment = require('moment');
require('tempusdominus-bootstrap-4');
``` doesn't render anything. it's just empty text field after initializing `$('#datetimepicker').datetimepicker()`

**Comment by chriseenberg** [comment]
Can i get back to you on that one, i just branched out and im unable to see my code as to what i did to make it work.

Kind regards Chris

**Comment by chriseenberg** [comment]
Okay, so i ended up downloading the library directly from github - adding the css and js file to my angular-cli.json then wrapped the datepicker in an angular component. I accessed the datepicker through jquery like so:
`$('#datetimepicker1').datetimepicker({
            allowInputToggle: true
        });`

If you have any specific questions to what might not work for you, i'll be happy to assist :)

**Comment by Eonasdan** [comment]
Sorry I don't know much about webpack. Can I close this? Did you guys figure it out?

**Comment by colonelclick** [comment]
I used npm (yarn) to add the package to webpack and then I was able to use these lines in my .js

```
import $ from 'jquery';
import 'tempusdominus-bootstrap-4';
```

The other sticking point for me was that my html was not quite correct, so I started with example code from the docs page to get it working and then modified it from there

**Comment by Zhorian** [comment]
I got it working by adding moment to plugins config 
```
var plugins = [
        new CleanWebpackPlugin(["./wwwroot/public/assets"]),
        new webpack.ProvidePlugin({
            $: "jquery",
            jQuery: "jquery",
            moment: "moment"
        })
    ];
```

**Comment by colonelclick** [comment]
Yes I think this can be closed. This was more of a webpack question, and multiple solutions/tips have been suggested here.
